src/base64url.c

```diff
@@ -71,7 +71,19 @@ ece_base64url_decode(const char* base64, size_t base64Len,
     }
   }
 
-  size_t maxDecodedLen = (base64Len * 3) / 4;
+  // Determine the size of the decoded result. Every 4 bytes of input decode
+  // to 3 bytes of output. The final quantum can be 2 or 3 bytes: 2 bytes
+  // decode to 1 byte, and 3 bytes decode to 2 bytes.
+  size_t maxDecodedLen = (base64Len / 4) * 3;
```

Comment:
Hmm, I'm not sure I want to make this a float. I *think* the flooring behavior is OK, but maybe not? If the length is 50, the string holds 12 groups of 4 + 2 at the end. Base64 decodes 4 to 3 and 2 to 1, so (12 * 3) + 1 = 37.